pkg/agent/nplagent/npl_agent_init.go

```diff
@@ -0,0 +1,76 @@
+// +build !windows
+
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nplagent
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/nplagent/k8s"
+	"github.com/vmware-tanzu/antrea/pkg/agent/nplagent/lib"
+	"github.com/vmware-tanzu/antrea/pkg/agent/nplagent/portcache"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/client-go/informers"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+
+	"github.com/vmware-tanzu/antrea/pkg/util/env"
+)
+
+// InitializeNPLAgent : start NodePortLocal (NPL) agent
```

Comment:
The convention for golang func comments is like:
InitializeNPLAgent starts NodePortLocal (NPL) agent.

And could you add ‘.’ at EOL for a comment sentence?